internal/provider/data_source_gitlab_repository_file_test.go

```diff
@@ -59,20 +60,24 @@ func testAccDataSourceGitlabRepositoryFile(src, n string) resource.TestCheckFunc
 }
 
 func testAccDataGitlabRepositoryFile(project string) string {
+	var executeFilemodeConfig string
+	if isExecuteFilemodeSupported, err := isGitLabVersionAtLeast(context.Background(), testGitlabClient, "14.10")(); err == nil && isExecuteFilemodeSupported {
```

Comment:
As an FYI for the future, you can use `context.TODO()` in cases where you need a context but do not have one. They're functionally identical, but `context.Background()` communicates a different intent.